entity-types/ext-triton/golden_metrics.yml

```diff
@@ -0,0 +1,48 @@
+nv_cpu_memory_total_bytes:
+  title: Provides the total CPU memory from the Triton inference server.
+  unit: BYTES
+  query:
+    select: latest(nv_cpu_memory_total_bytes)
+    from: Metric
+
+nv_cpu_memory_used_bytes:
```

Comment:
Does it help putting the unit in the metric name?

Reply:
The Triton Server metrics are presented in that manner.
Please refer to this document for metrics: https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/user_guide/metrics.html#cpu-metrics